packages/mystmd-py/scripts/pypi-deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+cp ../mystmd/README.md .
```

Comment:
This script should be run after releasing the JS mystmd library. It copies dependencies, updates the version in pypi to match npm, builds and deploys.